openapi/components/schemas/ReportJournal.yaml

```
@@ -0,0 +1,27 @@
+type: array
+items:
+  type: object
+  properties:
+    aggregationValue:
+      description: The aggregation field value.
+      type: string
+    bookedMonth:
+      description: Revenue booked month.
+      type: string
+      pattern: '^\d{4}-\d{2}$'
+      example: 2022-02
+    bookedAmount:
+      description: The revenue amount booked for recognition.
+      type: number
+      format: double
+      example: 25.0
+    recognizedAmount:
+      description: The revenue amount recognized.
```

nitpick: I find this easier to read
```suggestion
      description: Recognized revenue amount.
```